README.md

```diff
@@ -16,16 +16,19 @@ for details.
 
 * Thrustmaster T500RS
 * Thrustmaster TX
-
+* Thrustmaster TMX
 See the status of additional devices on
 [GitHub Projects](https://github.com/her001/tmdrv/projects/1).
 
 ## Requirements
 
 * [Python 3](https://www.python.org)
 * [python-libusb1](https://pypi.python.org/pypi/libusb1)
-* jscal (from [Linux Console](https://sourceforge.net/projects/linuxconsole),
-for calibration)
+* jscal 
+ - Ubuntu: sudo apt install joystick
+ - Other Distro's: compile from source from [Linux Console](https://sourceforge.net/projects/linuxconsole),
+for calibration) 
+
```

Comment:
I disagree with this change.

1. Nowhere else are there distro-specific instructions in the readme.
2. It isn't my fault that some distros (not just Ubuntu) don't have the name of the software in the package name nor the description.
3. Very few distros do not have Linux Console in their repositories; almost nobody will have to compile it if they don't want to.
4. A minor point, but the new line at the end must be removed.

Maybe this can be worded in a different way. Something like:
```markdown
* jscal (from [Linux Console](https://sourceforge.net/projects/linuxconsole), or `joystick` in some distros' package repositories, for calibration)
```